web/packages/plugins/ui/simpos-ui/src/sima/model/views/QuadCurrentCoeffPlot_src.js

```diff
@@ -3,7 +3,11 @@ import React from 'react'
 
 import Plot from 'react-plotly.js'
 
-const SIMA_Model_QuadCurrentCoeffPlot = ({ parent, document, children }) => {
+const SIMA_Model_QuadCurrentCoeffPlot = ({
+  parent,
+  document,
+  children,
+}: any) => {
   console.log(document)
```

Comment:
remove console.log?

Reply:
These are sintefs plugins. I try not to touch those. If they want to log, let them